config/custom_items.txt

```diff
@@ -508,4 +508,14 @@ item_path: /obj/item/clothing/suit/storage/toggle/bomber
 item_name: Combat Medic Jacket
 item_icon: khartling_bomber_open
 item_desc: A thick, well-worn leather jacket with a crisp green cross sewn on the back.
-}
\ No newline at end of file
+}
+
+{
+ckey: devildabeast
+character_name: Looping Song
+item_path: /obj/item/clothing/suit/storage/labcoat
+item_name: e UMi labcoat
+item_icon: labcoat_black_open
+item_desc: A suit that protects against minor chemical spills. Has a black stripe on the shoulder. The abbreviation \"e UMi\" is written on the back in bold text.
```

Comment:
BYOND won't respect your escape sequence when ingesting raw text like this. Just use plain quotation marks, like the other items. It splits at the : character and uses the character data on the RHS as is so you don't need to escape anything.